lintrules/draftcheck/rules.py

```diff
@@ -150,7 +150,7 @@ def check_no_space_before_cite(text, matches):
     return [m.span() for m in matches]
 
 
-#@rule(r'[^~]\\ref{')
+@rule(r'[^(~|{)]\\ref{')
```

Comment:
```suggestion
@rule(r'[^~{]\\ref{')
```
角括弧に含まれる文字以外にマッチだから、丸括弧の選択はなくても動くと思います

Reply:
承知しました
ありがとうございます！！

Reply:
丸括弧部分の修正と、\Eq{\ref{eq:shiki}}と\Fig{\ref{fig:hyou}}でエラーがでないことを確認しました！